### PR TITLE
Move to a single npm package

### PR DESCRIPTION
**Pull request by blink1073**
The npm package in the `juputerlab/` folder was adding complexity, disk space, and download time.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +68,7 @@
     "build:examples": "node scripts/buildexamples.js",
     "build:src": "tsc --project src && node scripts/copyfiles.js",
     "build:test": "webpack --config test/webpack.conf.js",
-    "build:serverextension": "cd jupyterlab && npm run build && cd ..",
+    "build:serverextension": "cd jupyterlab && webpack",
```
Comment (vidartf) [inline package.json]: Isn't the `&& cd ..` still needed?

Comment (blink1073) [inline package.json]: I just checked on OSX and Windows and the `cd ..` isn't required for any of them, so I removed the other uses in e0acbd6.

**Comment by jasongrout** [comment]
I'm okay with this as long as we still have a working example of a standalone JupyterLab package (so that people see how to easily customize JupyterLab by loading different plugins, etc.).

**Comment by blink1073** [comment]
You mean `examples/lab`?

**Comment by ellisonbg** [comment]
rebase, pls :)

**Comment by jasongrout** [comment]
> You mean examples/lab?

Yes, exactly. Thanks.

**Pull request merged**